lmdeploy/pytorch/kernels/cuda/pagedattention.py

```diff
@@ -1150,7 +1152,10 @@ def _get_block_d(Lk):
     is_decoding = q.shape[-3] == q_seqlens.size(0)
     if not is_decoding:
         BLOCK_DMODEL, BLOCK_DMODEL1, BLOCK_DV = _get_block_d(Lq)
-        BLOCK_M = max(16, min(BLOCK, 16384 // BLOCK_DMODEL))
+        if _nv_cap[0] < 8:
```

Comment:
Could `_nv_cap` be `None`?

Reply:
_nv_cap would be initialized at the begin of this function:

https://github.com/InternLM/lmdeploy/blob/3cfca58f387f375f4303256d868e2b5611d5a45f/lmdeploy/pytorch/kernels/cuda/pagedattention.py#L1104-L1108

`torch.cuda.get_device_capability()` would not be none I guess.